### PR TITLE
(cu 862j4cn7u) composable strategies withdrawal guide

### PR DESCRIPTION
**Pull request by Martin-Kiss**

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [CU-862j4cn7u Strategies Withdrawal Guide](https://app.clickup.com/t/862j4cn7u)

File: docs/docs/products/parachain-vault-strategy/composable-strategies-withdrawal-guide.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+![ethPerFarm_getAmountsOut](images-composable-strategies-withdrawal-guide/ethPerFarm-getAmountsOut.png)
+
+The value after the decimal point is the input value for field 5.b
+
+5. c) To get the tokensPerEth
+
+   In the first input field called `amountIn` input the value 1 * 10 at power of Decimal of TOKEN 
+   (18 decimals for DAI, 6 for USDC and 6 for USDT)
+
+   In the second field called path `address[]` you need to enter the first contract address of WET
```
Comment (0xBrainjar2) [inline docs/docs/products/parachain-vault-strategy/composable-strategies-withdrawal-guide.md:106]: im assuming WETH?

**Review by 0xBrainjar2 — COMMENTED** [review]
can we change all the pictures with red lines to be nice looking arrows instead of manually drawn red lines  - needs to be a bit more professional

**Comment by JafarAz** [comment]
@Martin-Kiss agreed with brainjar to merge for now and replace screenshots when Liviu is back

File: docs/docs/products/parachain-vault-strategy/composable-strategies-withdrawal-guide.md (hunk 1 above)
Comment (JafarAz) [inline docs/docs/products/parachain-vault-strategy/composable-strategies-withdrawal-guide.md:106]:
```suggestion
   In the second field called path `address[]` you need to enter the first contract address of WETH 
```

**Review by JafarAz — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by Dom-Roth — APPROVED** [review]

**Pull request merged**